### PR TITLE
Filtering block assets by module fails - Closes #1300 #1301

### LINKED ISSUE #1300: Filtering block assets by module fails
### Actual behavior
Filtering block assets by module fails.

Query: `/api/v3/blocks/assets?module=random`
Response: 
```
{
  "error": true,
  "message": "Server error: select count(`height`) as `count` from `blocks` where `height` is not null and (json_contains(`assetsModules`,'random')) - Invalid JSON text in argument 1 to function json_contains: \"Invalid value.\" at position 0."
}
```

### Expected behavior
The query should return a successful response.

### Steps to reproduce
- Query the endpoint with `module` request param

### Which version(s) does this affect? (Environment, OS, etc...)

`v0.7.0-alpha.1`

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #1300 #1301 

### How was it solved?
- [x] Fix `getBlocksByIDs` implementation
- [x] Fix regex for module name
- [x] Fix MySql whereJsonSupersetOf implementation
- [x] Publish framework to GitHub
- [x] Update all services dependencies

### How was it tested?
Local

### CODE REVIEW

File: framework/src/mysql.js [hunk 1]
```diff
@@ -243,9 +243,9 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 			const { property, values } = params.whereJsonSupersetOf;
 			query.where(function () {
 				const [val0, ...remValues] = Array.isArray(values) ? values : [values];
-				this.whereJsonSupersetOf(property, val0);
+				this.whereJsonSupersetOf(property, [val0]);
```
Comment (priojeetpriyom) [inline framework/src/mysql.js:246]: Can we rename `property` to `column` in respect to knex api?

File: services/gateway/shared/regex.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ const TOKEN_ID = /^\b[a-fA-F0-9]{16}\b$/;
 const BLS_KEY = /^\b[a-fA-F0-9]{96}\b$/;
 const PROOF_OF_POSSESSION = /^\b[a-fA-F0-9]{192}\b$/;
 const TRANSACTION = /^\b[0-9a-fA-F]+\b$/;
+const MODULE = /^(?:[a-z ]+,)*[a-z ]+$/;
```
Comment (sameersubudhi) [inline services/gateway/shared/regex.js:37]: Please update the regex for MODULE name.
```suggestion
const MODULE = /^\b(?:[\w!@$&.]{1,32}|,)+\b$/;
```

File: framework/src/mysql.js (hunk 1 above)
Comment (sameersubudhi) [reply framework/src/mysql.js:246]: This update will induce a lot of changes within the entire codebase. Let's skip this for now. We can look into this later.

Comment (priojeetpriyom) [reply framework/src/mysql.js:246]: We can just rename it here as atleast. When I was reading it, got an wrong idea thinking `property` is a key in the json. Later found it is actually the column name reading the knex doc

**Pull request merged**